src/policy/fees.cpp

```diff
@@ -710,6 +711,7 @@ CFeeRate CBlockPolicyEstimator::estimateRawFee(int confTarget, double successThr
 
 unsigned int CBlockPolicyEstimator::HighestTargetTracked(FeeEstimateHorizon horizon) const
 {
+    LOCK(cs_feeEstimator);
```

Comment:
I don't think we need exclusive locking for these operations, as `GetMaxConfirms` so concurrent access wouldn't result in incoherent data.